### PR TITLE
Adding more CaaS hosters

### PR DESCRIPTION
**Pull request by BretFisher**
This adds the CaaS term to the hosters section and a few more container hosters I found.

Bear Hugs!

bear-hug

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,9 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 # Where to start ?
 * [10-minute Interactive Tutorial](https://docs.docker.com/docker-for-mac/)
 * [Docker Training](http://training.docker.com/)
-* Read this complete article: [Basics – Docker, Containers, Hypervisors, CoreOS](http://etherealmind.com/basics-docker-containers-hypervisors-coreos/)
+* Read this complete article: [Basics – Docker, Containers,
```
Comment (veggiemonk) [inline README.md]: What happened here?

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Security](#security)
 	- [Serverless](#serverless)
 	- [Service Discovery](#service-discovery)
-	- [Services for running containers](#services-for-running-containers)
+	- [CaaS - Services for running containers](#services-for-running-containers)
```
Comment (veggiemonk) [inline README.md]: this will break links....

File: README.md [hunk 3]
```diff
@@ -365,8 +367,9 @@ Services to securely store your Docker images.
 * [proxy](https://github.com/factorish/proxy) - lightweight nginx based load balancer self using service discovery provided by registrator. by [@factorish](https://github.com/factorish)
 * [registrator](https://github.com/progrium/registrator) - Service registry bridge for Docker by [@progrium][progrium]
 
-## Services for running containers
+## CaaS (Containers as a Service) - Services for running/hosting containers
```
Comment (veggiemonk) [inline README.md]: If you change the name of the link, you have to change it also in the menu....
I'd recommend to see the result in github and test the link. 
Ping here if you need help 😄

File: README.md [hunk 4]
```diff
@@ -365,8 +367,9 @@ Services to securely store your Docker images.
...
 * [Amazon ECS](http://aws.amazon.com/ecs/) - A management service on EC2 that supports Docker containers.
+* [Arukas](https://arukas.io/) - Heroku-inspired CaaS
```
Comment (veggiemonk) [inline README.md]: Nice!

File: README.md [hunk 5]
```diff
@@ -378,6 +381,7 @@ Services to securely store your Docker images.
 * [IBM Bluemix](https://console.ng.bluemix.net/) - Run Docker containers in a hosted cloud environment on IBM Bluemix.
 * [OpenShift Dedicated](https://www.openshift.com/dedicated/index.html) - A hosted [OpenShift][openshift] cluster for running your Docker containers managed by Red Hat.
 * [Orchard](https://www.orchardup.com/) (part of Docker Inc) - Get a Docker host in the cloud, instantly.
+* [Sloppy.io](https://sloppy.io/) - all-in-one solution for container deployment and hosting – made and hosted in Germany
```
Comment (veggiemonk) [inline README.md]: Thanks for your contribution 😄

**Comment by veggiemonk** [comment]
I would give extra point for bear hug but github is limited as of the kind of reaction I can give ;-)

File: README.md (hunk 2 above)
Comment (BretFisher) [inline README.md]: Not sure how will it break links? it's not changing link, just title of link.

File: README.md (hunk 3 above)
Comment (BretFisher) [inline README.md]: ah right, duh, got it.

Comment (BretFisher) [inline README.md]: ok reverted this, leaving the CaaS mention in the ToC rather then here. good eye.

File: README.md (hunk 1 above)
Comment (BretFisher) [inline README.md]: fixed.

**Comment by BretFisher** [comment]
ok how 'bout now?

File: README.md (hunk 3 above)
Comment (veggiemonk) [inline README.md]: Sorry I should explain better. The modification of the title here must also be applied in the ToC. #services-for-running-containers ===> #caas--containers-as-a-service---services-for-running-containers so something like that....

Comment (BretFisher) [inline README.md]: right, no worries, that should all be fixed in latest commit.

**Pull request merged**